npc/re/merchants/eden_market.txt

```diff
@@ -42,8 +42,7 @@ function	script	F_CoolDown	{
 
 -	marketshop	para_key10	FAKE_NPC,7026:100000:10,7027:100000:10 // todo
 
--	marketshop	para_wp00	FAKE_NPC,1181:10000000:1
-
+-	marketshop	para_wp00	FAKE_NPC,1181:10000000:1,1231:10000000:1,13039:10000000:1,1140:10000000:1,1142:10000000:1,1478:10000000:1,1306:10000000:1,1137:10000000:1,16033:10000000:1,1648:10000000:1,2004:10000000:1,18117:10000000:1,1814:10000000:1,1584:10000000:1,2113:10000000:1
```

Comment:
Updating to reflect suggestions below
```suggestion
-	marketshop	para_wp00	FAKE_NPC,1181:10000000:1,1231:10000000:1,13039:10000000:1,1140:10000000:1,1422:10000000:1,1478:10000000:1,1306:10000000:1,1377:10000000:1,16033:10000000:1,1648:10000000:1,2004:10000000:1,18117:10000000:1,1816:10000000:1,1584:10000000:1,1265:10000000:1
```

Reply:
Apologies mate, really scrappy overall. Corrected and updated.